src/createServer.js

```diff
@@ -1,12 +1,22 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { userRouter } = require('./routes/usersRoute');
+const { clearAllUsers } = require('./services/usersService');
+const { expensesRouter } = require('./routes/expensesRouter');
+const { clearExpenses } = require('./services/expensesService');
 
 function createServer() {
-  // Use express to create a server
-  // Add a routes to the server
-  // Return the server (express app)
-}
+  const app = express();
+
+  clearAllUsers();
+  clearExpenses();
+
+  app.use('/users', express.json(), userRouter);
+  app.use('/expenses', express.json(), expensesRouter);
```

Comment:
```suggestion
  app.use(express.json());
  app.use('/users', userRouter);
  app.use('/expenses', expensesRouter);
```
You can add `express.json()` midleware for the app itself